RGPosadas/LitterRally
# TASK-86: Misc Cleanup

closes #314

How to test:
- Make sure the following points are addressed from the ticket:
    - 86.1 On Discover/Upcoming/My Events, show “No events found” when queries are empty
    - 86.2 Smaller Mull logo and nav bar on desktop
    - 86.3 Remove all back buttons when on desktop
        - There are exceptions that must be made for this. Namely whenever the onclick is overridden, then I don't hide the button, since it might be used for something other than a browser-type back button
    - 86.4 All buttons and inputs should have feedback when you hover and click
    - 86.5 Remove all share buttons
    - 86.6 All text input limits should match backend varchar limits
    - 86.7 Remove Map button and move the buttons around according to this [mockup](https://www.figma.com/file/FJPXMYI4ULCnuCEZvPs67k/Mull?node-id=3691%3A87700)
